.github/workflows/multi-arch-test-build.yml

```
@@ -64,6 +64,9 @@ jobs:
       - name: Determine branch name
         run: |
           BRANCH="${GITHUB_BASE_REF#refs/heads/}"
+          if [[ "$BRANCH" != "master" && "$BRANCH" != "main" && ! "$BRANCH" =~ ^openwrt-[0-9]+\.[0-9]+$ ]]; then
+            BRANCH="master"
+          fi
```

Perhaps this is a bit more readable:
```suggestion
          case "$BRANCH" in
            main|master|openwrt-[0-9]*\.[0-9]*)
              ;;
            *)
              BRANCH="master"
              ;;
          esac
```

Although the default should be repo-specific, I imagine.

---

Hmm. That's indeed better! :) Thank you for your suggestion. I will apply it shortly.